apps/server/src/terrain.ts

```diff
@@ -0,0 +1,49 @@
+import * as fs from 'fs';
+
+import axios from 'axios';
+
+import * as path from 'path';
+import * as os from 'os';
+
+
+const SIMBRIDGE_FOLDER = path.join(os.homedir() + '/flybywire-externaltools-simbridge');
```

Comment:
Why not some friendlier like `FlyByWireSim/SimBridge`? :D We can put other things under FlyByWireSim in future if we need.

Reply:
See https://github.com/nodejs/node-gyp/issues/1124 and their solution https://github.com/nodejs/node-gyp/commit/2b5ce52a26018692e89377b0504e7efa3b4900b6#diff-af2728b0ae9aeb2df1f7acd883eb34a8b116ab3bb7576c1c2ea0848fe2c7a0e4R5

Reply:
Sounds good 👍

Reply:
Or on second thought, %LOCALAPPDATA% is hidden on windows per default I believe. There might be some confusion if people want to put pdfs or coroutes there I guess.

Reply:
Opted for the Documents directory now (for user data, stuff like terrain map and dependencies stay in the install folder for now..)